telosnetwork/telos-wallet
# #200 | Epic - Wrap/Unwrap

# Fixes #200

## Description
This feature branch PR comprises all of the work from #440, #441, #442, and #443. 

## Test scenarios
- go to https://deploy-preview-530--wallet-staging.netlify.app/
- log in with an account that has TLOS available
- click the overflow menu button on the **TLOS** row in the wallet balance tab
    - it should have an item 'Wrap' which should link to the Wrap page
- click the overflow menu button on the **WTLOS** row in the wallet balance tab
    - it should have an item 'Unwrap' which should link to the Wrap page - unwrap tab
- click the overflow menu button on any other row in the wallet balance tab
    - it should not have an item 'Unwrap' or 'Wrap'
- click the wrap link in the site nav
     - the page should match [the design](https://www.figma.com/file/eIpbnt6vqwqptEbZWtwO8Y/v0.3?type=design&node-id=364-2714&mode=design&t=NSq6quC6Hzmsvc1l-0)
- try wrapping TLOS
    - you should be able to wrap successfully
    - the balance son the page should update once the tx is complete (check the header and the amount above the currency input)
- go to the unwrap tab
    - page should match [the design](https://www.figma.com/file/eIpbnt6vqwqptEbZWtwO8Y/v0.3?type=design&node-id=364-2721&mode=design&t=NSq6quC6Hzmsvc1l-0)
- unwrap some TLOS
    - you should be able to unwrap successfully
    - the balance son the page should update once the tx is complete (check the header and the amount above the currency input)
- repeat these steps on iOS (safari, metamask app, safepal app), android (chrome, metamask app, safepal app), and desktop (metamask, safepal)


## Checklist:
